flaml/nlp/result_analysis/azure_utils.py

```diff
@@ -340,43 +333,53 @@ def _get_complete_connection_string(self):
                + self._azure_key + ";EndpointSuffix=core.windows.net"
 
     def _init_azure_clients(self):
+        from azure.storage.blob import ContainerClient
         connection_string = self._get_complete_connection_string()
-        container_client = ContainerClient.from_connection_string(conn_str=connection_string,
-                                                                  container_name=self._container_name)
-        return container_client
+        try:
+            container_client = ContainerClient.from_connection_string(conn_str=connection_string,
+                                                                      container_name=self._container_name)
+            return container_client
+        except ValueError:
+            print("Container name not specified")
```

Comment:
How do you know that's the cause?

Reply:
In this line, the error cannot be due to connection_str, changed to "_container_name is unspecified or wrongly specified, please specify _container_name in AzureUtils"

Reply:
Please go ahead and make the change.

Reply:
Please go ahead and make the change.